nmap-service-probes

```diff
@@ -15473,3 +15473,22 @@ Probe TCP VersionRequest q|VERSION|
 ports 3310
 rarity 8
 match clam m|^ClamAV ([\w.]+)/(\w+)/(.+)$| p/ClamAV/ v/$1 ($2)/ i/AV definitions updated on:$3/
+
+##############################NEXT PROBE##############################
+# RSE for IBM Explorer for z/OS (FMID HALG300)
+Probe TCP RSE q|ZmFrZTpmYWtl0\x00\x30\x00|
+rarity 9
+ports 4035
+sslports 4035
```

Comment:
Yes. The system this runs on uses AT-TLS which allows them to enable SSL for the port. This is the recommended action, but the port doesn't change.